cernopendata/modules/fixtures/data/records/cms-derived-csv-2012.json

```diff
@@ -0,0 +1,69 @@
+{
+  "abstract": {
+    "description": "<p>This document contains Higgs candidate events that pass the selections found in the example Higgs-to-four-lepton analysis of CMS public data.</p> <p>These data were selected for use in education and outreach and contain a subset of the total event information. They are not suitable for a full physics analysis.</p>"
+  },
+  "accelerator": "CERN-LHC",
+  "authors": [
+    {
+      "name": "McCauley, Thomas",
+      "orcid": "FIXME"
+    }
+  ],
+  "collections": [
+    "CMS-Derived-Datasets"
+  ],
+  "date_published": "2019",
+  "distribution": {
+    "formats": [
+      "csv",
+      "ig",
+      "json"
+    ]
+  },
+  "doi": "10.7483/OPENDATA.CMS.FIX.ME",
```

Comment:
Maybe leave DOI fully empty for now, so that we avoid clashes.  We can always add one later.

Reply:
I "reserved" a specific one; shall I remove it? It's not minted yet.

Reply:
If you reserved, please use the real one.... While we shall deploy on DEV only for now, we can already use the target one...